### DISCUSSION TITLE
MultiSelect组件不按选中先后显示

### DISCUSSION THREAD

**Discussion by csharpunreal** [dotnetcore/BootstrapBlazor#4333] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[Demo.zip](https://github.com/user-attachments/files/17054864/Demo.zip)
在EditTemplate模板中时，选中项会自动按items集合顺序排序，而不是按项目选中先后排序，不作为表单模板时没有此问题，请大佬指教

### Expected Behavior

如何让MultiSelect在EditTemplate模板中按照选中先后排序而非按照items顺序排序

### Interactive render mode

Interactive Server (Interactive server-side rendering (interactive SSR) using Blazor Server)

### Steps To Reproduce

[Demo.zip](https://github.com/user-attachments/files/17054906/Demo.zip)

### .NET Version

net8

**Comment by bb-auto** [comment]
@csharpunreal Thank you for reporting. We will give triage later.

**Comment by Vision-Zhang** [comment]
把 MultiSelect 组件包一层成为一个新的组件放到 EditTemplate 中

**Comment by ArgoZhang** [comment, marked as answer]
@csharpunreal 这不是组件的 bug 是你没有认真看文档，代码我更新了。请下载下面的压缩包，认真比对

[Demo.zip](https://github.com/user-attachments/files/17071194/Demo.zip)
